### PR TITLE
Migrated to use towncrier for change logs

### PR DESCRIPTION
**Pull request by andy-maier**
You can display the change log for the upcoming version 1.8.0 by running:
```
make develop
towncrier build --draft
```
with this branch checked out.

### CODE REVIEW

File: docs/development.rst [hunk 1]
```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
 .. _How to Get Your Change Into the Linux Kernel: https://www.kernel.org/doc/Documentation/SubmittingPatches
 
 
+.. _`Making a change`:
+
+Making a change
```
Comment (KSchopmeyer) [inline docs/development.rst:544]: suggest that we change this to "Creating and submitting a change to pywbem" . This title was so short it was hard to understand.  Same for pywbemtools.

File: docs/development.rst [hunk 2]
```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
...
+---------------
+
```
Comment (KSchopmeyer) [inline docs/development.rst:546]: Add the following:   All changes to pywbem are made through github with PRs created on topic branches and merged with the current master after successful group review.

File: docs/development.rst [hunk 3]
```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
...
+You can review how your changes will show up in the final change log for
+the upcoming release by running:
+
+.. code-block:: sh
+    towncrier build --draft
+
+Always make sure that your pushed branch has either just one commit, or if you
+do multiple things, one commit for each logical change. What is not OK is to
+keep the possibly multiple commits it took you to get to the final result for
```
Comment (KSchopmeyer) [inline docs/development.rst:631]: Modify to change "possibly mulitiple" to  "submit for review a PR with multiple " ..

File: docs/development.rst [hunk 4]
```diff
@@ -539,6 +539,99 @@ For further discussion of good and bad practices regarding commits, see:
...
+
+Making a change
+---------------
+
+To make a change, create a topic branch. You can assume that you are the only
+one using that branch, so force-pushes to that branch and rebasing that branch
+is fine.
+
+When you are ready to push your change, describe the change for users of the
+package in a change fragment file. To create a change fragment file, execute:
```
Comment (KSchopmeyer) [inline docs/development.rst:552]: should we define further wha change fragment file is an the link to towncrier.

**Review by KSchopmeyer — CHANGES_REQUESTED** [review]
We have both DEVELOP.md and docs/development.rst.   Would it make sense to just point one at the other rather than duplicatng the descriptions.  I see where in the long run this will take more work but will be more consistent eventually and hopefully lead to fewer minor changes that have to be made during the push/review process. 

Response by Andy: The content is not duplicated between the two files. Having said that, it does make sense to integrate DEVELP.md into docs/development.rst (in a separate PR).

File: docs/development.rst (hunk 2 above)
Comment (andy-maier) [reply docs/development.rst:546]: DONE as proposed.

File: docs/development.rst (hunk 3 above)
Comment (andy-maier) [reply docs/development.rst:631]: DONE as proposed.

File: docs/development.rst (hunk 4 above)
Comment (andy-maier) [reply docs/development.rst:552]: DONE. Added short explanation and link to towncrier.

**Comment by andy-maier** [comment]
I resolved all the review comments.

**Pull request merged**